controllers/factory.go

```diff
@@ -213,14 +213,18 @@ func (f *Factory) withVSphereClusterReconciler() *Factory {
 }
 
 func (f *Factory) withSnowClusterReconciler() *Factory {
-	f.withCNIReconciler()
+	f.withCNIReconciler().withTracker()
```

Comment:
What is this?

Reply:
tracker? You can see it here https://github.com/aws/eks-anywhere/blob/079ca2f5b838e81b9f7446519b0e895cc3205bf1/controllers/factory.go#L132-L155

It builds a remote client tracker. It builds kube clients for workload clusters from kubeconfigs stored in secrets.